app/assets/javascripts/invite_users.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Also, it is important to be able to write the files that you need yourself, it helps to pull back the curtain and reveal the magic.